.github/workflows/php.yml

```
@@ -97,26 +99,22 @@ jobs:
   unit:
     name: 'Run unit tests'
     if: "!contains(github.event.commits[0].message, '[skip ci]') && !contains(github.event.commits[0].message, '[ci skip]')"
-    runs-on: 'ubuntu-18.04'
+    runs-on: 'ubuntu-20.04'
     env:
       BEDITA_API_KEY: 12345
       BEDITA_API: http://127.0.0.1:8080
       BEDITA_ADMIN_USR: admin
       BEDITA_ADMIN_PWD: admin
       BEDITA_DOCKER_IMG: bedita/bedita:4.2.1
```

```suggestion
      BEDITA_DOCKER_IMG: bedita/bedita:4.6.1
```
more recent docker image version